handler/router.go

```diff
@@ -18,25 +20,20 @@ func NewHandler(services *service.Service, logger logging.Logger) *Handler {
 
 func (h *Handler) InitRoutes() *gin.Engine {
 	router := gin.New()
+	router.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler))
 
-	router.Use(
-		CorsMiddleware,
-	)
-
-	auth := router.Group("/auth")
+	role := router.Group("/roles")
 	{
-		auth.POST("/sign-in", h.signIn)
+		role.POST("/", h.createRole)
+		role.POST("/roleToPerms", h.bindRoleWithPerms)
+		role.GET("/:id", h.getRoleById)
+		role.GET("/", h.getAllRoles)
 	}
-
-	api := router.Group("/api", h.userIdentity)
+	perm := router.Group("/perms")
 	{
-		lists := api.Group("/roles")
-		{
-			lists.POST("/", h.createRole)
-			lists.POST("/permission", h.createPermission)
-			lists.POST("/roleToPermission", h.createRoleToPermission)
-			lists.GET("/:id", h.getRoleById)
-		}
+		perm.POST("/", h.createPerm)
+		perm.GET("/:id", h.getPermsByRoleId)
```

Comment:
When you have route like `/permissions/123` everybody would assume that 123 is permission id, not role. To load permissions for role use similar convention to what I've mentioned above `GET roles/123/permissions`